Negative comparatives and a bit of vocab (#4)
Agma-Schwa/ULTRAFRENCH: GRAMMAR.tex

%% ‘lẹ-’: from PF ‘plus’. Comparative prefix.
\subsection{Comparison}\label{subsec:comparison}

> Unlike in many other languages, there are 8 comparatives in UF, 4 positive and 4 negative: The affirming comparatives, so called
> because it affirms the comparison (‘better, and also good’, ‘less hot, and also not very hot’); the denying comparative, which denies the comparison
> (‘better, but not good’, ‘less hot, but still hot’), the sufficient comparative, which indicates that there is just enough of something,
> and the neutral comparative, which does not make any statement about the comparison (‘better’, ‘less hot’).

Reviewer: ```suggestion
Unlike in many other languages, there are 8 comparatives in UF, 4 positive and 4 negative: The affirming comparatives, so called
because they affirm the comparison (‘better, and also good’, ‘less hot, and also not very hot’); the denying comparatives, which deny the comparison
(‘better, but not good’, ‘less hot, but still hot’), the sufficient comparatives, which indicate that there is just (not) enough of something,
and the neutral comparatives, which do not make any statement about the comparison (‘better’, ‘less hot’).
```

To illustrate the difference between the three: We might say that an ant is ‘bigger’ than a grain of sand, but
an ant is still not big, all things considered. By contrast, an elephant may be ‘smaller’ than a mountain,
but that doesn’t mean that an elephant is small. Finally, a human is small enough to fit through a regular-sized
door.

> In UF, the comparatives are expressed by four infixes, which are prefixed directly to the stem. For positive comparatives, the affirming

Reviewer: ```suggestion
In UF, the comparatives are expressed by a total of eight infixes, which are prefixed directly to the stem. For positive comparatives, the affirming
```

comparative prefix is \w{lẹ}, the denying comparative prefix is \w{y’ŷ}, the sufficient comparative prefix is \w{ah},
and the neutral comparative prefix is \w{rê}. Thus, we have \w{ḅẹt’hâ} ‘small’, \w{lẹḅẹt’hâ} ‘smaller, and also small’,
\w{y’ŷḅẹt’hâ} ‘smaller, but not small’, \w{ahḅẹt’hâ} ‘small enough’, and \w{rêḅẹt’hâ} ‘smaller’. Unlike other degrees
of comparison, the sufficient comparative is always regular.

> For negative comparatives, the affirming comparative prefix is \w{laû}, the denying comparative prefix is \w{y’aû}, the sufficient
> comparative prefix is \w{haû} and the neutral comparative prefix is \w{raû}. Those are formed from a contraction of the positive prefixes and the
> negative marker \w{aû}.

Reviewer: ```suggestion
For negative comparatives, the affirming comparative prefix is \w{laû}, the denying comparative prefix is \w{y’aû}, the sufficient
comparative prefix is \w{haû}, and the neutral comparative prefix is \w{raû}. Diachronically, these resulted from a contraction of the positive prefixes and the
nominal negative marker \w{aû}.
```

neutral comparative. Note that \w{ré} lenites, whereas \w{râdvâ} does not. Thus, we have \w{rébhẹt’hâ} or
\w{râdvâḅẹt’hâ} ‘smallest’.

> The negative superlative is formed with the prefix \w{râdvaû}, which is formed from \w{râdvâ} and \w{aû} similarly to the comparatives.

Reviewer: ```suggestion
The negative superlative is formed with the prefix \w{râdvaû}, which diachronically derives from \w{râdvâ} and \w{aû}, similarly to the negative comparatives.
```
